janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server.yaml

```diff
@@ -17,7 +17,7 @@ port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
 graphs: {
-  graph: conf/gremlin-server/janusgraph-cql-es-server.properties
```

Comment:
I think we should think of removing all different gremlin-server.yaml and just have a single the user just have to decide between `janusgraph-*.properties` files. This would be a major breaking change.

Besides, this is also a breaking change and should be mentioned in the changelog.

Reply:
Agree. We should think about removing them but I think we should not spend much time on the current release and think about adding those changes in future releases.
I have added a not to the changelog

Reply:
If you change this behaviour here, you have to update janusgraph.sh. The janusgraph.sh uses the gremlin-server.yaml and expects cql-es is started.

Reply:
I think the best solution for this PR would be to let `janusgraph.sh` start `gremlin-server.sh` explicitly with a `cql-es` YAML. Since cql-es was the default until now, there is no YAML for that. So, we should create another one and then use that in `janusgraph.sh`.
That way, users of `janusgraph.sh` wouldn't notice a difference and `gremlin-server.sh` can work without CQL and ES in the truncated distribution.